patroni/ctl.py

```diff
@@ -646,3 +646,54 @@ def configure(config_file, dcs, namespace):
     config['dcs_api'] = str(dcs)
     config['namespace'] = str(namespace)
     store_config(config, config_file)
+
+
+def touch_member(config, dcs):
+    ''' Rip of the ha.touch_member without inter-class dependencies '''
+    p = Postgresql(config['postgresql'])
```

Comment:
I could just extract the code that creates the connection and api strings out of it, but there are dependencies in it, and I wanted to avoid copying too much code (a part of touch_member from HA looked small and easy to comprehend to just copy it here directly).